release/release-notes/data-prepper.release-notes-2.8.0.md

```diff
@@ -62,7 +59,6 @@
 * [BUG] Empty DLQ entries when version conflicts occur ([#4301](https://github.com/opensearch-project/data-prepper/issues/4301))
 * [BUG] otel sources should show a more clear exception when receiving data that cannot be processed based on the configured compression type ([#4022](https://github.com/opensearch-project/data-prepper/issues/4022))
 * [BUG] : unable to set field_delimiter_regex ([#2946](https://github.com/opensearch-project/data-prepper/issues/2946))
-* [BUG]Incorrect Behavior of Obfuscate Processor with Predefined Pattern "%{CREDIT_CARD_NUMBER}" [#4340](https://github.com/opensearch-project/data-prepper/issues/4340)
```

Comment:
This one should stay.

Reply:
@dlvenable it is there. I removed a duplicate line.